develop/dev-guide-bookshop-schema-design.md

```diff
@@ -4,71 +4,91 @@ title: Bookshop Example Application
 
 # Bookshop Example Application
 
-Bookshop is a virtual online bookstore application, you can easily buy books of various categories in Bookshop, and you can also rate on the books you have read.
+Bookshop is a virtual online bookstore application through which you can buy books of various categories and rate the books you have read.
+
+This document describes the following information:
```

Comment:
You can refer to new changes on the chinese document, our website already has a page navigation feature, no need to add redundant outlines.

https://github.com/pingcap/docs-cn/blob/e02b2eab472ac67ab2a9d470dcf2e00447204cc8/develop/dev-guide-bookshop-schema-design.md#bookshop-%E5%BA%94%E7%94%A8